test/e2e/runner/load.go

```diff
@@ -54,9 +54,19 @@ func Load(ctx context.Context, testnet *e2e.Testnet) error {
 		case numSeen := <-chSuccess:
 			success += numSeen
 		case <-ctx.Done():
-			if success == 0 {
+			// if we couldn't submit any transactions,
+			// that's probably a problem and the test
+			// should error; however, for very short tests
+			// we shouldn't abort.
+			if success == 0 && time.Since(started) < 2*time.Second {
 				return errors.New("failed to submit any transactions")
 			}
+
+			// TODO test networks should be able to
```

Comment:
Could you please add an issue number to this?

Reply:
I softened the language for this, but I don't think it's ready to be an issue: It's work covered/discussed by the e2e RFC, and I think we should probably not do it unless there were other features that we wanted to engage with, and I think the current implementation in this area is consistent with current requirements.